integrations/dbt/bigquery/setup.cfg

```diff
@@ -11,14 +11,14 @@ serialize =
 search = version="{current_version}"
 replace = version="{new_version}"
 
-[bumpversion:file:./setup.py]
-search = marquez-integration-common=={current_version}
```

Comment:
Does this module not require the `marquez-integration-common` package?

Reply:
No, it was copied over by mistake. Here are the dependencies in [`setup.py`](https://github.com/MarquezProject/marquez/blob/fix/versioning-for-dbt-integrations/integrations/dbt/bigquery/setup.py):

```
requirements = [
    "dbt-core>=0.20.0b1",
    "dbt-bigquery>=0.20.0b1",
    "sqlparse>=0.3.1",
    "openlineage-python==0.0.1rc6"
]
```

Reply:
I see. I guess the "reusable" bigquery utilities aren't that reusable, then?

Reply:
That was my initial thinking as well. But, with the DBT integration being experimental, and using _adopters_ it most likely differs greatly from our airflow integration. Perhaps @mobuchowski could provide more insight?